### PR TITLE
[PDI-16884] - Cassandra Output step does not work with apache-cassand…

### PR DESCRIPTION
**Pull request by YuryBY**
…ra-2.1.0

@mbatchelor, could you please take a look?

### CODE REVIEW

**Comment by luckyxiang2017** [comment]
HTTPPOST，line 286，please add encode
if ( entity != null ) {
                            String encode = data.realEncoding;
                            if (StringUtil.isEmpty(encode)) {
                                ContentType contentType = ContentType.get(entity);
                                encode = contentType == null ? "UTF-8" : contentType.getCharset().name();
                            }

                            body = new String(EntityUtils.toByteArray(entity), encode);
                        } else {
                            body = "";
                        }

**Comment by luckyxiang2017** [comment]
You upgrade and change the original function，it's bad

File: core/src/main/java/org/pentaho/di/core/plugins/KettleURLClassLoader.java [hunk 1]
```diff
@@ -104,7 +104,7 @@ public String getName() {
   protected synchronized Class<?> loadClass( String arg0, boolean arg1 ) throws ClassNotFoundException {
     try {
       return loadClassFromThisLoader( arg0, arg1 );
-    } catch ( ClassNotFoundException | NoClassDefFoundError e ) {
+    } catch ( ClassNotFoundException | NoClassDefFoundError | SecurityException e ) {
```
Comment (mbatchelor) [inline core/src/main/java/org/pentaho/di/core/plugins/KettleURLClassLoader.java]: Would prefer logging here - need to log the exception somewhere. Swallowing it is not a good fix.

**Comment by YuryBY** [comment]
@mbatchelor, PR have updated

**Pull request merged**